tools/src/main/scala/scala/scalanative/linker/Reach.scala

```diff
@@ -183,11 +191,33 @@ class Reach(config: build.Config, entries: Seq[Global], loader: ClassLoader) {
             ()
         }
       case info: Trait =>
+        // Register given trait as a subtrait of
+        // all its transitive parent traits.
         def loopTraits(traitInfo: Trait): Unit = {
           traitInfo.subtraits += info
           traitInfo.traits.foreach(loopTraits)
         }
         info.traits.foreach(loopTraits)
+
+        // Initialize default method implementations that
+        // can be resolved on a given trait. It includes both
+        // all of its parent default methods and any of the
+        // non-abstract method declared directly in this trait.
+        info.linearized.foreach {
+          case parentTraitInfo: Trait =>
+            info.responds ++= parentTraitInfo.responds
+          case _ =>
+            util.unreachable
+        }
+        loaded(info.name).foreach {
+          case (_, defn: Defn.Define) =>
+            val Global.Member(_, sig) = defn.name
+
+            info.responds(sig) = defn.name
+
+          case _ =>
```

Comment:
I suggest removing the blank lines here:
```suggestion
            info.responds(sig) = defn.name
          case _ =>
```